xcode/Subconscious/Shared/Components/Common/Story/StoryUserView.swift

```diff
@@ -47,26 +47,34 @@ struct StoryUserView: View {
     var body: some View {
         VStack(alignment: .leading, spacing: 0) {
             HStack(alignment: .center, spacing: AppTheme.unit2) {
-                ProfilePic(pfp: story.user.pfp, size: .medium)
-                PetnameView(
-                    nickname: story.user.nickname,
-                    petname: story.user.address.petname
-                )
-                .fontWeight(.medium)
-                .foregroundColor(.accentColor)
-                
-                Spacer()
-                
-                switch (story.isFollowingUser, story.user.category) {
-                case (true, _):
-                    Image.from(appIcon: .following)
-                        .foregroundColor(.secondary)
-                case (_, .you):
-                    Image.from(appIcon: .you(colorScheme))
-                        .foregroundColor(.secondary)
-                case (_, _):
-                    EmptyView()
+                Group {
+                    ProfilePic(pfp: story.user.pfp, size: .medium)
+                    PetnameView(
+                        nickname: story.user.nickname,
+                        petname: story.user.address.petname
+                    )
+                    .fontWeight(.medium)
+                    .foregroundColor(.accentColor)
+                        
+                    Spacer()
+                    
+                    if !story.isResolved {
+                        Image(systemName: "exclamationmark.triangle.fill")
+                            .foregroundColor(.secondary)
+                    } else {
+                        switch (story.isFollowingUser, story.user.category) {
+                        case (true, _):
+                            Image.from(appIcon: .following)
+                                .foregroundColor(.secondary)
+                        case (_, .you):
+                            Image.from(appIcon: .you(colorScheme))
+                                .foregroundColor(.secondary)
+                        case (_, _):
+                            EmptyView()
+                        }
+                    }
                 }
+                .disabled(!story.isResolved)
```

Comment:
I can't disable the entire view because it would block the `Menu` from opening (which contains the shortcut to unfollow)